lib/pry/input_completer.rb

```diff
@@ -239,4 +227,39 @@ def build_path(input)
     end
     return path, input
   end
+
+  private
+
+  def self.fetch_obj_space_collection
```

Comment:
Yes, I see. Sorry fro my comment before I've checked the code. I've added a not in https://github.com/pry/pry/pull/1583#issuecomment-260182876